index.html

```diff
@@ -45,6 +45,106 @@ <h1>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+          <div>
+            <div class="image-box"></div>
+            <div class="text-box">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <div class="tech">
+                <span>Ruby on rails</span>
+                <span>css</span>
+                <span>html</span>
+                <span>Javascript</span>
+              </div>
+              <div class="see-project"><a href="#">See Project</a></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about">
+        <!-- <img src="images/illustration-about-me-mobile.svg" alt="" /> -->
+        <div class="about-content">
+          <h1>About me</h1>
```

Comment:
- [ ] Kindly use any other `hx` tag instead of the `h1` tag here. The `h1` tag can only use once as the header of website here.